test/integration/internal-extension.js

```diff
@@ -100,14 +100,17 @@ test('load sync', t => {
 
     t.equal(vm.runtime._blockInfo.length, 1);
 
-    // blocks should be an array of two items: a button pseudo-block and a reporter block.
-    t.equal(vm.runtime._blockInfo[0].blocks.length, 2);
+    // blocks should be an array of two items: a button pseudo-block, a reporter block, and a command block.
```

Comment:
Rephrasing the comment like this might make it harder to accidentally miss updating the number in the comment
```suggestion
    // blocks should be an array of [button pseudo-block, reporter block, command block].
```